### PR TITLE
Fix some typos and some minor improvements

### PR DESCRIPTION
**Pull request by Cmdu76**

### CODE REVIEW

**Comment by me-shaon** [comment]
@arguiot Could you please review it?

File: translations/LICENSE_fr-FR [hunk 1]
```diff
@@ -1,11 +1,11 @@
 License publique GLWT (Bonne Chance Avec Ça)
 (c) Tout droits réservés au monde entier, à l'exception du ou des auteurs.
 
-Chacun est autorisé à copier, distribuer, modifier, fusionner, vendre, publier, sous-licencier 
-ou faire tout ce qu'il désire avec ce logiciel, mais à ses PROPRES RISQUES ET PERILS.
+Chacun est autorisé à copier, distribuer, modifier, fusionner, vendre, publier, 
+sous-licencer ou ce qu'il désire avec ce logiciel, mais à ses PROPRES RISQUES ET PERILS.
```
Comment (arguiot) [inline translations/LICENSE_fr-FR]: Something isn't right with this sentence, we should keep the older one.

File: translations/LICENSE_fr-FR [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-L'auteur n'a aucune idée de ce que fait le code source de ce projet.
-Il fonctionne peut être, ou pas du tout, il n'y a pas d'autres options.
+L'auteur n'a absolument aucune idée de ce que fait le code de ce projet.
```
Comment (arguiot) [inline translations/LICENSE_fr-FR]: Not necessary, but it makes sense here

File: translations/LICENSE_fr-FR [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+Il fonctionne peut-être, ou pas du tout. Il n'y a pas d'autres options.
```
Comment (arguiot) [inline translations/LICENSE_fr-FR]: This modification is ok for me

File: translations/LICENSE_fr-FR [hunk 4]
```diff
@@ -15,7 +15,7 @@ Il fonctionne peut être, ou pas du tout, il n'y a pas d'autres options.
 TRACE PERMETTANT DE RETROUVER L'AUTEUR du projet originel pour l'accuser 
 ou le tenir responsable.
 
-LE OU LES AUTEUR(S) NE POURRONT EN AUCUN CAS ETRE TENUS REPONSABLES POUR DES 
+LE OU LES AUTEUR(S) NE POURRONT EN AUCUN CAS ETRE TENUS RESPONSABLES POUR DES
```
Comment (arguiot) [inline translations/LICENSE_fr-FR]: Fixes a typo, modification is fine

File: translations/LICENSE_fr-FR (hunk 1 above)
Comment (Cmdu76) [reply translations/LICENSE_fr-FR]: Hum yes you're right... Sorry

**Comment by me-shaon** [comment]
@Cmdu76 please resolve the comments of @arguiot as soon as you fixed them. I'll then merge your PR. :)

**Comment by me-shaon** [comment]
@Cmdu76 Thanks for your contribution.

**Pull request merged**